packages/graphile-build/src/plugins/NodePlugin.js

```diff
@@ -60,6 +67,13 @@ export default (function NodePlugin(
               JSON.stringify([this.getNodeAlias(Type), ...identifiers])
             );
           },
+          getNodeIdFromTypeAndIdentifiers(nodeId) {
```

Comment:
Wait; isn't this `getTypeAndIdentifiersFromNodeId` rather than `getNodeIdFromTypeAndIdentifiers`